tests/test_dd.py

```diff
@@ -44,6 +45,15 @@ def test_B1855(self):
             np.abs(pint_resids_us.value - self.ltres) < 1e-7
         ), "DD B1855 TEST FAILED"
 
+    def test_stand_alone_model_params_updates(self):
```

Comment:
Should this just iterate over `*.par`? That is, any reason not to test that all models correctly import all parameters? One would just trivially report success for non-binary models.

Reply:
Is this needed any more now that all par files are tested?